### PR TITLE
Change synapse prefetching default to none.

### PR DESCRIPTION
**Pull request by hernando**

### CODE REVIEW

File: tests/brain/python/circuit.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def test_load_morphology(self):
 
     def test_metypes(self):
         count = self.circuit.num_neurons()
-        mnames = self.circuit.morphology_names()
+        names = self.circuit.morphology_names()
```
Comment (tribal-tec) [inline tests/brain/python/circuit.py]: sure?

**Comment by tribal-tec** [comment]
remove the brain::SynapsePrefetch::none from the unit tests

**Comment by hernando** [comment]
I've replaced none with all which is the case that was not covered after the default parameter was changed.

**Comment by tribal-tec** [comment]
Hmm, I didn't mean that. In the C++ tests which have explicit 'none', those ones can be changed to remove the now obsolete 'none' and use 'all' for those which rely on the not-true anymore 'all'.

**Comment by hernando** [comment]
I thought you were referring to the Python tests.

**Comment by hernando** [comment]
ping

**Pull request merged**